### PR TITLE
Fix for 500px.com

### PR DESCRIPTION
**Pull request by salvatore-cirasa**
Fixed fav icon in dark mode.

### CODE REVIEW

**Comment by Gusted** [comment]
Hi @salvatore-cirasa!

Could you point me where these fixes are needed for on the site?

Regards,
Gusted

**Comment by salvatore-cirasa** [comment]
Hi @Gusted,
yes! In popular/editors photos page (es. https://500px.com/popular), in bottom right corner of every photo, there is an heart for insert in fav list. With dark theme this icon has the same aspect in on/off mode.

I'm not expert of this project, if you've a better solution with _darkreader_ classes, feel free to discard my changes and use yours.

Regards
Salvo

P.S. Thank you for this tool, saved my eyes!

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -216,6 +216,19 @@ div.description > span {
 
 ================================
 
+500px.com
+
+CSS
+.button.new_fav .icon .shape {
+    fill: transparent !important;
+    stroke: rgb(232, 230, 227);
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:224]:
```suggestion
```
The stroke value is already correctly modified by dark reader.

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -216,6 +216,19 @@ div.description > span {
...
+}
+.button.only_icon.hearted .icon .shape, .button.only_icon.liked .icon .shape {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:226]: I'm not able to find these elements, I assume they are only available for people that are signed in?

Comment (salvatore-cirasa) [reply src/config/dynamic-theme-fixes.config:226]: Yes, you can add photos in fav list only if signed in. For test try to add `hearted` class in a like button:

```html
<div class="like-button">
  <div>
    <a class="button new_fav only_icon hearted">
      <span class="value"></span>
      <svg class="icon" version="1.1" viewBox="-6.9 -13.1 40 40" x="0px" y="0px">
        <path class="shape" d="" fill="#FFFFFF" style="--darkreader-inline-fill: #e8e6e3;" data-darkreader-inline-fill=""></path>
      </svg>
    </a>
  </div>
</div>
```

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (salvatore-cirasa) [reply src/config/dynamic-theme-fixes.config:224]: Sorry, I lost this value in my tests.

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:226]: In that case..

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -216,6 +216,19 @@ div.description > span {
...
+    fill: var(--darkreader-inline-fill) !important;
+}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:228]:
```suggestion

IGNORE INLINE STYLE
.button.only_icon .icon .shape
```

Comment (salvatore-cirasa) [reply src/config/dynamic-theme-fixes.config:228]: Hi @Gusted, sorry I'm late.
I have tested your solution and it is better than mine.
I will remember it for the next pull requests!

**Pull request closed without merging**